src/bioregistry/data/bioregistry.json

```diff
@@ -4133,14 +4133,15 @@
   "biorxiv": {
     "description": "The bioRxiv is a preprint server for biology",
     "example": "2021.07.23.453588",
+    "pattern": "^\\d{4}\\.\\d{2}\\.\\d{2}\\.)?\\d{6}(v\\d{1,3})?$",
```

Comment:
Regarding the regex pattern, I allowed version suffixes up to `v999`. Assuming a preprint will never have 1000 versions. Possibly 100 would be safe, but not in a future where preprints are updated automatically on a scheduled basis.

Am worried that the main 6 digit number will eventually be exceeded. A recent ID is `2022.01.24.477526v1`, suggesting that there have been `477526` preprints and eventually this might exceed 1 million.

Should we keep this at 6 for now and fix in the future, or proactively allow up to 7 or 8 digits?

Reply:
I think it’s alright to just have an arbitrary \d in this spot

Reply:
But then again it’s nice to have extra guarantees for funky looking data, so maybe just go one order of magnitude higher than what’s there already

Reply:
Made 6-8 digits in https://github.com/biopragmatics/bioregistry/pull/293/commits/78dd4e2e3664df7c7a3ecad04174027d5f65a4ac. Seems like if we're to hit 1 million preprints, we could also soon hit 10 million.

By the way, the smallest ID I could find for a biorxiv preprint is `000091`. I do like catching errors like `biorxiv:91` which will fail to resolve.

Reply:
> By the way, the smallest ID I could find for a biorxiv preprint is 000091.

Looking through the Nov 2013 archives, there is also `000067`